models/Recipe.model.js

```diff
@@ -2,7 +2,43 @@ const mongoose = require('mongoose');
 const Schema = mongoose.Schema;
 
 const recipeSchema = new Schema({
-  // TODO: write the schema
+
+  title: {
+    type: String,
+    unique: true,
+    required: true
+  },
+  level: {
+    type: String,
+    enum: ['Easy Peasy', 'Amateur Chef', 'UltraPro Chef']
+  },
+  ingredientes: {
+    type: [String]
+  },
+  cuisine: {
+    type: String,
+    required: true
+  },
+  dishType: {
+    type: String,
+    enum: ['breakfast', 'main_course', 'soup', 'snack', 'drink', 'dessert', 'other']
+  },
+  image: {
+    type: String,
+    default: "https://images.media-allrecipes.com/images/75131.jpg"
+  },
+  duration: {
+    type: Number,
+    min: 0
+  },
+  creator: {
+    type: String
+  },
+  created: {
+    type: Date,
+    default: Date.now
+  }
+
```

Comment:
- `now` es un método de `Date`, por lo que tienes de llamarlo como función:

```suggestion
default: Date.now()
```